source/customizations.rst

```diff
@@ -599,6 +599,11 @@ connection alive is not factored into this.
 of activity (in milliseconds). After this duration, the connection will be closed
 regardless of activity. Its default is 3600000 milliseconds (1 hour).
 
+``OOD_SHELL_TERM`` allows the default ``TERM=xterm-16color`` to be overriden. Should only
+be set to ones whose escape codes are fully supported by the underlying
+`_hterm_ library <https://chromium.googlesource.com/apps/libapps/+/HEAD/hterm/docs/ControlSequences.md>`__.
```

Comment:
While the actual link may be OK, I think you're trying to make it _italics_ with the `_<>_` ? I don't think we should do that - it renders as literal `_hterm_`.

<img width="1277" height="190" alt="image" src="https://github.com/user-attachments/assets/260789ee-25ec-4a30-8d78-7c8445ed6bef" />

Reply:
Thanks. Force pushed a revision without the `_<>_`.